### ISSUE TITLE
import kipoiseq automatically run some unrelated python scripts / rerun unrelated jobs in log?

### ISSUE THREAD

**Issue by princeofpotato** [kipoi/kipoi#634]
Environment:
Anaconda
Python 3.8
kipoi                     0.7.3                    pypi_0    pypi
kipoi-conda               0.2.2                    pypi_0    pypi
kipoi-utils               0.6.0                    pypi_0    pypi
kipoiseq                  0.5.2                    pypi_0    pypi
(the problem also shows for other versions of kipoiseq)

Issue:
When  "import kipoiseq",  some unrelated python scripts automatically run!! 
I tested a python file with only 1 line of code:  "import kipoiseq" and the problem still shows. Recreate conda environment doesn't solve the problem.

The jobs which automatically run are some python scripts that previously run on the same environment (my previous works). So I think maybe "import kipoiseq" recall the jobs somewhere like a log file?

**Comment by haimasree** [comment]
Hello! Thanks for reporting. Just so I understand the problem -  if you import kipoiseq it is looking for some log file? Do you know which log files? I dont see any error n the nightly tests for python 3.8 so I am not too sure whats going on. Is there a problem with running kipoi or is it degraded performance you are worried about?

**Comment by princeofpotato** [comment]
Thanks for your reply. I’ve found where the problem happens now. It’s about kipoi. When “import kipoi”, the code will finally goes to a script which import many different models where one of them is “test”. Because of it, it tries to find a script called “test.py” in a different folder(one of my another project) to run.  I’m not sure if it’s about the working log or local models files for my kipoi. But maybe you can see it happens if you try to create a file called “test.py” in the same folder with the script which includes “import kipoi “.

**Comment by haimasree** [comment]
Sorry but I still dont think I understand your setup or the error you are encountering. Is it test or tests ? There is a folder called [tests](https://github.com/kipoi/models/tree/master/tests) and this should be inside your local model repo as well (typically [here](~/.kipoi/config.yaml). Does changing test.py to something else say dummy.py or something like that get rid of the error?

**Comment by princeofpotato** [comment]
Thanks for your attention. I found where the script calls the "test" model but don't know what it tries to do, like whether to call the local tests model. Because it failed to do so.
What I said about test.py is located in the same folder as where I write the python script.
Emmm, like:
-my python working dir
--main.py
--test.py

Within the main.py:
import kipoi

Within the test.py:
print("I'm test")

If I run main.py, it will show "I'm test" for me.


The reason: I found that when "import kipoi" runs, it finally goes to:
envs/my_conda_env_name/lib/python3.8/site-packages/future/standard_library/__init__.py
At the bottom of this file, it runs __import__(m, level=0) and one value of m is "test".
It seems like trying to call a model named "test*" but recognise my test.py as that model incorrectly?
